packages/expo-dev-menu/ios/OverlayContainer/Internal/Extensions/UIScrollViiew+Utils.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  UIScrollViiew+Utils.swift
+//  Pods
+//
+//  Created by Gaétan Zanella on 28/11/2018.
+//
+
+import UIKit
+
+extension UIScrollView {
+
+    var scrollsUp: Bool {
+        return panGestureRecognizer.yDirection == .up
+    }
+
+    var isContentOriginInBounds: Bool {
+        topOffsetInContent <= 0.0
+    }
+
+    var topOffsetInContent: CGFloat {
+        contentOffset.y + oc_adjustedContentInset.top
+    }
+
+    func scrollToTop() {
+        contentOffset.y = -oc_adjustedContentInset.top
+    }
+}
+
+
+extension UIScrollView {
+    
+    var oc_adjustedContentInset: UIEdgeInsets {
+        if #available(iOS 11.0, *) {
+            return self.adjustedContentInset
+        } else {
```

Comment:
I don't believe we support anything earlier than 12.4, or maybe 13.0 now, so not sure this is needed.

Reply:
It's not our code and I don't want to modify that.